models/players.js

```diff
@@ -38,6 +49,26 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var fetchTeamColors = function (league, team_name){
+  var colorsObject = _.first(team_colors_nba.team_colors);
+  if (league == "nba"){
+    for (i=0;i<colorsObject.teams.length;i++){
+      if (colorsObject.teams[i].team_name == team_name){
+        var colors = [];
```

Comment:
save a line of code by defining colors at the top of the function. Also its a little more readable when the variables are defined at the start